### PR TITLE
Hyper-v: Added STOR-VSS-BACKUPRESTORE-MULTIFS-VHDX test

### PR DESCRIPTION
**Pull request by prashantmhetar**

### CODE REVIEW

File: Testscripts/Windows/AddVhdxHardDisk.ps1 [hunk 1]
```diff
@@ -207,8 +207,26 @@ function Main {
         return $False
     }
 
+    # Lisav2 framework supports the param with unique name.
+    # Modifying the input SCSI_ and IDE_ multiple params and rebuilding testParams
     $params = $testParams.TrimEnd(";").Split(";")
     foreach ($p in $params)
+    {
```
Comment (lizzha) [inline Testscripts/Windows/AddVhdxHardDisk.ps1]: why not put them in the same foreach ($p in $params) loop of line 229

Comment (prashantmhetar) [reply Testscripts/Windows/AddVhdxHardDisk.ps1]: This loop has been kept separate as it modifies the input params and reassembles the input params. This newly created params are passed to the second loop.

Comment (lizzha) [reply Testscripts/Windows/AddVhdxHardDisk.ps1]: but they can be put in a same loop

Comment (prashantmhetar) [reply Testscripts/Windows/AddVhdxHardDisk.ps1]: Modified the code to use same loop

File: Testscripts/Windows/AddVhdxHardDisk.ps1 [hunk 2]
```diff
@@ -211,8 +211,16 @@ function Main {
...
+        $var = $fields[0].Trim()
+        if ($var -match "SCSI_") {
+            $var = "SCSI"
+        }
+        elseif ($var -match "IDE_") {
+            $var = "IDE"
+        }
+        switch ($var)
         {
         "rootDIR"   { $rootDir = $fields[1].Trim() }
```
Comment (chvalean) [inline Testscripts/Windows/AddVhdxHardDisk.ps1]: param not used/required

**Comment by chvalean** [comment]
@lizzha please sign-off on this and merge the PR

**Pull request merged**